src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/EnumConverterFactory.cs

```diff
@@ -62,5 +41,26 @@ public static JsonConverter Create(Type enumType, EnumConverterOptions converter
                 parameterTypes: [typeof(EnumConverterOptions), typeof(JsonNamingPolicy), typeof(JsonSerializerOptions)],
                 parameters: [converterOptions, namingPolicy, options])!;
         }
+
+        internal static class Helpers
```

Comment:
Nice trick. Could you add a comment explaining why you're nesting these methods inside another type?